app/src/main/res/drawable/ic_tutorial_img_2.xml

```diff
@@ -0,0 +1,11 @@
+    <path android:fillColor="#878787" android:pathData="M16,86.87c-0.86,-1.57 -1.64,-3 -2.38,-4.38a1,1 0,0 0,-1.32 -0.57c-2,0.56 -4.09,1 -6.39,1.58L7.4,78.12 0,76.78l5.28,-3.91 -5.21,-4 7.4,-1.3L6.06,62.22c2,0.5 4,0.87 5.81,1.47a1.52,1.52 0,0 0,2.21 -0.95c0.6,-1.26 1.35,-2.45 2.13,-3.83 1.38,1.27 2.71,2.45 4,3.69a1.2,1.2 0,0 0,1.67 0.25c1.71,-0.93 3.47,-1.78 5.38,-2.75 0.19,1.52 0.41,2.91 0.52,4.32 0.07,0.85 0.36,1.16 1.25,1.13 2.12,-0.07 4.23,0 6.45,0l-3.37,4.66L38.69,73 32,75.68l3.39,4.73H27.86l-0.78,5.32c-1.87,-1 -3.58,-1.8 -5.25,-2.72a1.3,1.3 0,0 0,-1.79 0.21C18.78,84.44 17.45,85.59 16,86.87Z"/>
+    <path android:fillColor="#878787" android:pathData="M22,51.06l3.7,7.67 -1.83,0.93c-1.74,-3.58 -3.45,-7.1 -5.15,-10.62a14.08,14.08 0,0 1,-1 -2.31,3.33 3.33,0 0,1 1.74,-4q8.66,-4.26 17.35,-8.42c1.9,-0.91 3.84,-0.08 4.85,2 2.36,4.81 4.68,9.63 7,14.44q4.7,9.64 9.37,19.29c1.42,2.92 0.81,4.69 -2.11,6.11Q48,80 40,83.84A3.57,3.57 0,0 1,35 82.35h2.16c0.71,0 1.42,0.07 2.49,0.13l-4.29,-6c1.89,-0.8 3.74,-1.5 5.51,-2.37 4.35,-2.15 8.67,-4.38 13,-6.56 0.77,-0.39 0.35,-0.85 0.15,-1.28q-2.93,-6 -5.86,-12.07L42.52,42.66c-0.19,-0.39 -0.4,-0.77 -0.65,-1.26ZM43.59,75.64a3,3 0,0 0,2.78 3A2.66,2.66 0,0 0,49 75.84a3,3 0,0 0,-2.84 -2.94A2.66,2.66 0,0 0,43.59 75.64ZM33.76,41.15a1.32,1.32 0,0 0,-1.95 -1.38c-1.62,0.75 -3.21,1.56 -4.82,2.35 -0.74,0.36 -1.14,0.91 -0.73,1.72a1.2,1.2 0,0 0,1.8 0.56c1.6,-0.8 3.23,-1.54 4.81,-2.37A3.54,3.54 0,0 0,33.76 41.15Z"/>
+    <path android:fillColor="#878787" android:pathData="M154.39,76.68l3.38,6c1.9,3.38 4.69,4.64 8.52,4.07 7,-1 13.91,-2 20.86,-3a10.8,10.8 0,0 0,3.33 -1.11c-0.29,-0.17 -0.58,-0.36 -0.88,-0.5l-8,-3.88c-4,-1.93 -4.85,-3.57 -4.09,-8.08l-3.07,0.43a39.81,39.81 0,0 0,-4.38 0.63c-1.27,0.32 -1.82,-0.1 -2.42,-1.18 -3.28,-6 -6.65,-11.84 -10,-17.77A10.81,10.81 0,0 0,150 46.67c-4.85,-1 -9.66,-2.11 -14.5,-3.15a11.82,11.82 0,0 0,-8.25 1.18c-8.21,4.47 -16.32,9.12 -24.47,13.71A7,7 0,0 0,99.67 62c-3.74,9.2 -7.51,18.4 -11.19,27.63a7.39,7.39 0,0 0,13.72 5.51c3.31,-8 6.52,-16 9.82,-24.05a3.72,3.72 0,0 1,1.41 -1.63c1.9,-1.18 3.88,-2.23 5.84,-3.32 0.33,-0.19 0.68,-0.34 1.25,-0.63 -0.71,3.23 -1.35,6.22 -2,9.21 -2.08,9.5 -4.26,19 -6.19,28.52a81.22,81.22 0,0 0,-1.12 10.23c-0.27,3.79 -0.34,7.59 -0.55,11.38a2.57,2.57 0,0 1,-0.62 1.36q-9.56,11.4 -19.16,22.76c-2,2.3 -2.85,4.79 -1.9,7.74a7.35,7.35 0,0 0,12.47 2.69c2.53,-2.78 4.89,-5.72 7.31,-8.6 4.43,-5.26 8.74,-10.62 13.33,-15.74a13.51,13.51 0,0 0,3.42 -8.81c0.18,-3.46 0.61,-6.91 1,-10.57l4.49,0.87c0.7,0.14 1.4,0.36 2.11,0.43a9.47,9.47 0,0 1,6.11 3.27c4.56,4.78 9.28,9.4 14,14a4.56,4.56 0,0 1,1.44 2.89c1,8 2,16.08 2.92,24.13 0.47,4 3,6.74 6.79,7.23a7.18,7.18 0,0 0,7.53 -4.74,11.62 11.62,0 0,0 0.37,-4.7c-0.77,-7 -1.69,-13.95 -2.56,-20.92 -0.32,-2.5 -0.59,-5 -1,-7.49a7.53,7.53 0,0 0,-2.36 -4.21c-5.92,-5.89 -11.81,-11.83 -17.76,-17.71a2.45,2.45 0,0 1,-0.78 -2.61c2.14,-9.45 4.2,-18.92 6.29,-28.38C154.09,77.48 154.21,77.23 154.39,76.68Z"/>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/8aa95bc22269ef11a970c62e715e84ca8f84ab7e/app/src/main/res/drawable/ic_tutorial_img_2.xml#L7"></span>Very long vector path (1420 characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector.</td>
    </tr>
  </tbody>
</table>